src/Game.cpp

```diff
@@ -12,8 +12,12 @@ std::string Game::getName() const
 
 int Game::score() const
 {
-    //needs implementation after made convertValueToPairOfIntsFrame()
-    return 0;
+    int partialScore = 0;
+    for (int i=0; i<10; i++)
```

Comment:
@anorberciak you can write it in the correct formatting 
`for (int i = 0; i < 10; i++)`

Reply:
Done